### PR TITLE
Change path to fsPath

### LINKED ISSUE #1: Error: <semantic> TypeScript Server Error (4.7.3)
It doesn't work anywhere. Hover provider works, but ts request seems to be failing, here is what I see in `Log (Extension Host)`:

```console
[2022-06-21 04:42:57.106] [exthost] [error] Error: <semantic> TypeScript Server Error (4.7.3)
No Project.
Error: No Project.
    at Object.ThrowNoProject (c:\Users\Vitaly\AppData\Local\Programs\Microsoft VS Code\resources\app\extensions\node_modules\typescript\lib\tsserver.js:167622:23)
```

I've tried different TS version.

image

### PR DESCRIPTION
**Pull request by pablomayobre**
**TL;DR:** Using path resulted in errors for tsserver in non-unix platforms due to the tsserver not finding the specified file.

To debug this issue (#1), I got logs from tsserver by following [these instructions](https://stackoverflow.com/a/39758878) (and the ones in the official [tsserver docs](https://github.com/microsoft/TypeScript/wiki/Standalone-Server-%28tsserver%29#logging)), opened vscode and then tried to use the extension on a TypeScript project.

> **Note**
> I would share the logs I got but unfortunately they contain sensitive information such as file paths and project names. 

The issue was that the path the extension was sending to tsserver was something like `/c:/Users/...` when, in Windows, they should look like `c:\\Users\...`.

[`fsPath`](https://code.visualstudio.com/api/references/vscode-api#Uri:~:text=The%20difference%20to%20the%20path%2Dproperty) handles drive letters and directory separators across different platforms.

---------------------

Fixes #1

### CODE REVIEW

**Comment by orta** [comment]
Lovely fix and great explanation!

**Comment by orta** [comment]
INFO  Publishing 'Orta.vscode-twoslash-queries v1.0.2'...
 INFO  Extension URL (might take a few minutes): https://marketplace.visualstudio.com/items?itemName=Orta.vscode-twoslash-queries
 INFO  Hub URL: https://marketplace.visualstudio.com/manage/publishers/Orta/extensions/vscode-twoslash-queries/hub
 DONE  Published Orta.vscode-twoslash-queries v1.0.2.
